### DISCUSSION TITLE
Step 7 in Data loading wizard for production planning

### DISCUSSION THREAD

**Discussion by atistabiur** [frePPLe/frepple#533] [Q&A]
Program is no allowing to checkbox 5 and 6 in spite i have already completed these tasks.
What is preventing to got to step 7 after completing all the steps before ??

**Comment by jdetaeye** [comment]
Step 5 and 6 are locked as long as you don't have manufacturing order or purchase order yet.    Which is indeed too strict.
I'll relax that to keep it locked as long as operations (step 3) and item suppliers (step 4) aren't defined yet. 

As a workaround, go to the screen "purchasing/purchase orders" and create a record there in the status "proposed". That will unlock the steps and allow you to go forward.

**Reply by atistabiur** [reply]
thanks
